docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -132,7 +139,7 @@ for asset in assets:
 ```
 
 ## Exporting a whole project
-There is also a method to export the whole project into specific export formats. It can be useful when your goal is to use one of the standard output formats.
+Kili has method to export the whole project into specific export formats. It can be useful when your goal is to use one of the standard output formats.
```

Comment:
One of:
- has a method
- is able
- allows you to export

Reply:
Typo again. Sorry. Fixed.